pgsql/CMakeLists.txt

```diff
@@ -16,11 +20,16 @@ set ( PC_INSTALL_EXENSIONS
   "${PROJECT_BINARY_DIR}/pgsql/pointcloud.control"
   )
 
-configure_file(
-    pointcloud.sql.in
-    "${PROJECT_BINARY_DIR}/pgsql/pointcloud--${POINTCLOUD_VERSION}.sql"
+configure_file(pointcloud.sql.in pointcloud.sql)
+configure_file(li3ds.sql.in li3ds.sql)
```

Comment:
And I guess my patch is a step towards this, by isolating the li3ds specific code.